scripts/SANS/sans/algorithm_detail/batch_execution.py

```diff
@@ -172,23 +181,55 @@ def load_workspaces_from_states(state):
 # Function for plotting
 # ----------------------------------------------------------------------------------------------------------------------
 def plot_workspace(reduction_package, output_graph):
+    """
+    Plotting continuous output when on MantidPlot
+    This function should be deleted if and when MantidPlot is no longer a part of Mantid
+
+    :param reduction_package: An object containing the reduced workspaces
+    :param output_graph: Name to the plot window
+    :return: None
+    """
     if reduction_package.reduction_mode == ISISReductionMode.All:
-        graph_handle = mantidplot.plotSpectrum([reduction_package.reduced_hab, reduction_package.reduced_lab], 0,
-                                               window=mantidplot.graph(output_graph), clearWindow=True)
+        graph_handle = plotSpectrum([reduction_package.reduced_hab, reduction_package.reduced_lab], 0,
+                                               window=graph(output_graph), clearWindow=True)
         graph_handle.activeLayer().logLogAxes()
     elif reduction_package.reduction_mode == ISISReductionMode.HAB:
-        graph_handle = mantidplot.plotSpectrum(reduction_package.reduced_hab, 0, window=mantidplot.graph(output_graph), clearWindow=True)
+        graph_handle = plotSpectrum(reduction_package.reduced_hab, 0, window=graph(output_graph), clearWindow=True)
         graph_handle.activeLayer().logLogAxes()
     elif reduction_package.reduction_mode == ISISReductionMode.LAB:
-        graph_handle = mantidplot.plotSpectrum(reduction_package.reduced_lab, 0, window=mantidplot.graph(output_graph), clearWindow=True)
+        graph_handle = plotSpectrum(reduction_package.reduced_lab, 0, window=graph(output_graph), clearWindow=True)
         graph_handle.activeLayer().logLogAxes()
     elif reduction_package.reduction_mode == ISISReductionMode.Merged:
-        graph_handle = mantidplot.plotSpectrum([reduction_package.reduced_merged,
+        graph_handle = plotSpectrum([reduction_package.reduced_merged,
                                                 reduction_package.reduced_hab, reduction_package.reduced_lab], 0,
-                                               window=mantidplot.graph(output_graph), clearWindow=True)
+                                               window=graph(output_graph), clearWindow=True)
         graph_handle.activeLayer().logLogAxes()
 
 
+def plot_workspace_workbench(reduction_package, output_graph):
```

Comment:
Can we call this `plot_workspace_matplotlib`. The idea is that the interfaces will not rely on workbench at all so we don't want there to appear to be a dependence.